docs/samples/browser-plugin-meetings/app.js

```diff
@@ -933,6 +951,31 @@ function getMediaStreams(mediaSettings = getMediaSettings(), audioVideoInputDevi
     return Promise.reject(new Error('No valid meeting object.'));
   }
 
+  if (isMultistream) {
+    console.log('MeetingControls#getMediaStreams() :: using webrtc-core local tracks')
+
+    localMedia.microphoneTrack = await webex.meetings.mediaHelpers.createMicrophoneTrack({...audioVideoInputDevices.audio});
```

Comment:
internal we can use the mediaHelper but when exporting to developers cant we map webex.meetings.createMicrophoneTrack ?. not sure if we discussed about this before